torch/quantization/quantize.py

```diff
@@ -198,14 +219,27 @@ def quantize(model, run_fn, run_args):
     model.eval()
     model = prepare(model)
     run_fn(model, run_args)
-    convert(model)
+    convert(model, mapping)
+    return model
+
+def quantize_dynamic(model, qconfig_dict=None, mapping=DEFAULT_DYNAMIC_MODULE_MAPPING):
+    r"""Converts a float model to dynamic quantized model. Do dynamic training and output a quantized model.
+    """
+    model.eval()
+    propagate_qconfig(model, qconfig_dict)
+    convert(model, mapping)
+    return model
+
+def prepare_qat(model, mapping=DEFAULT_QAT_MODULE_MAPPING):
+    model = prepare(model)
+    model = convert(model, mapping)
     return model
 
-def quantize_qat(model, run_fn, run_args):
+def quantize_qat(model, run_fn, run_args, mapping=DEFAULT_QAT_MODULE_MAPPING):
```

Comment:
we actually have two mappings in qat, and this change seems unrelated, revert?

Reply:
Reverted.